themes/digital.gov/src/js/common/in-page-navigation.js

```diff
@@ -0,0 +1,8 @@
+document.addEventListener("DOMContentLoaded", (event) => {
+  let relatedItems = document.querySelector(".content__related-items");
+  let inPageNav = document.querySelector(".usa-in-page-nav__nav");
+
+  if (relatedItems) {
+    relatedItems.before(inPageNav);
+  }
```

Comment:
Alternatively we could return early if there aren't related items.

```js
if (!relatedItems) {
  return;
}

relatedItems.before(inPageNav);
```

Reply:
Do we also want to check the length and hide `nav.usa-in-page-nav__nav` if there are less than 3?

Reply:
I've updated the code to hide `in-page nav` with less than 3 items. Like the simpler logic you suggested!